js/src/Bars.ts

```diff
@@ -814,7 +814,7 @@ export class Bars extends Mark {
             })
             .selectAll('.bar')
             .style("fill", fill ? this.get_mark_color.bind(this) : "none")
-            .style("stroke", stroke ? stroke : this.get_mark_color.bind(this))
+            .style("stroke", stroke ? stroke : "none")
```

Comment:
It's another heterogeneity here. The `fill` attribute being a boolean, I would have expected the `stroke` attribute to be a boolean as well. And I would expect both the fill and the stroke to behave the same when choosing their colors.

You PR is definitely an improvement (even a bug fix) in the current bqplot implementation, but we might be able to make some more improvements on the API.